changelog.d/5734.bugfix

```diff
@@ -0,0 +1 @@
+Fix servlet metric names.
```

Comment:
could you enlarge on this a little? Are the metrics in question prometheus metrics or some other kind? How are they broken? When was the brokenness introduced?

Reply:
How's that?